modules/pel/peltool/peltool.py

```diff
@@ -211,29 +211,41 @@ def severityBasedExclusion(uh: UserHeader, config: Config) -> bool:
             return False
     return True
 
-def excludePEL(uh: UserHeader, config: Config) -> bool:
+def includePEL(uh: UserHeader, config: Config) -> bool:
     """
     Evaluates whether a PEL meets criteria based on specified conditions.
-    Returns True if the entry should be Excluded, False otherwise.
+    Returns True if the entry should be Included, False otherwise.
     """
-    if config.critSysTerm and uh.eventSeverity != SeverityValues.critSysTermSeverity.value:
+    if config.critSysTerm and uh.eventSeverity == SeverityValues.critSysTermSeverity.value:
         return True
-    
-    if config.non_serviceable_only and uh.isServiceable():
+
+    if config.non_serviceable_only and not uh.isServiceable():
         return True
     
-    if config.serviceable_only and not uh.isServiceable():
+    if config.serviceable_only and uh.isServiceable():
+        return True
+
+    if config.severities and not severityBasedExclusion(uh, config):
+        return True
+
+    if config.hidden and uh.actionFlags & ActionFlagsValues.hiddenActionFlag.value:
         return True
     
-    if config.severities:
-        if severityBasedExclusion(uh, config):
-            return True
-    
-    if not config.hidden:
-        if uh.actionFlags & ActionFlagsValues.hiddenActionFlag.value:
-            return True
+    """
```

Comment:
It doesn't quite match [my sample code comment](https://github.com/ibm-openbmc/openpower-pel-parsers/pull/57#discussion_r1652553130), but I understand you're waiting to clarify the difference between hidden and non-serviceable to decide on the LN245 condition.

Reply:
Will work on isHidden function in user_head in future patch.